core/src/main/java/com/scalar/db/api/Scan.java

```diff
@@ -291,6 +297,24 @@ public Scan withProjections(Collection<String> projections) {
     return (Scan) super.withProjections(projections);
   }
 
+  Scan withConjunctions(Collection<Conjunction> conjunctions) {
+    this.conjunctions.addAll(conjunctions);
+    return this;
+  }
```

Comment:
I misunderstand something, but this method is package private. Is "deprecated" still necessary? I imagined current public withFooBar() mutation methods would be changed to package private ones like this function in v5.0.0.

Reply:
@brfrn169 Just a confirmation. Will those mutation methods themselves be deleted in v5.0.0? In addition, for example, `Scan` will have `ScanBuilder`, and the builder can directly access the private member of `Scan` without using `withFooBar()`. That's why we should make it deprecated even if a package private method, is my guess correct?

Reply:
Understood. Since the method in question is package-private and not public, there's no need to remove it or mark it as deprecated. Let's keep it as it is. If further changes or considerations are needed in the future, we can revisit this decision.